core/arch/arm/plat-imx/conf.mk

```diff
@@ -1,11 +1,20 @@
 PLATFORM_FLAVOR ?= mx6ulevk
 
-ifeq ($(PLATFORM_FLAVOR),mx6ulevk)
-arm32-platform-cpuarch		:= cortex-a7
+# Get SoC associated with the PLATFORM_FLAVOR
+ifeq ($(PLATFORM_FLAVOR),$(filter $(PLATFORM_FLAVOR),mx6ulevk))
```

Comment:
Well, my goal here and for two of the commented sections below, was to have a coherent syntax for all i.MX6 families and ease the process to add a board as the test is already written and you only need to add the board name to the list. 

Buts yes, looking back to it after a few days, maybe I have been going to far here.